apps/site/app/[locale]/not-found.tsx

```diff
@@ -19,13 +19,15 @@ const NotFoundPage: FC = () => {
       <main>
         404
         <h1 className="special -mt-4">{t('layouts.error.notFound.title')}</h1>
-        <div className="my-4 flex items-center justify-center">
-          <Image
-            src="/static/images/node-mascot.svg"
-            alt="The Node.js mascot"
-            height={114.69}
-            width={100}
-          />
+        <div className="my-4 flex h-[300] items-center justify-center">
```

Comment:
I'm not in love with this extra `h-300` class here, but it helped simplify the css greatly. There is no more translate x,y logic. 

One improvement would be to occupy less space on smaller viewports. I tried `sm:h-[150]` and it wouldn't work.